.clang-format

```diff
@@ -1,57 +1,82 @@
-AccessModifierOffset: -2
-AlignEscapedNewlinesLeft: false
-AlignTrailingComments: false
-AllowAllParametersOfDeclarationOnNextLine: false
-AllowShortFunctionsOnASingleLine: true
-AllowShortIfStatementsOnASingleLine: false
-AllowShortLoopsOnASingleLine: false
-AlwaysBreakBeforeMultilineStrings: false
-AlwaysBreakTemplateDeclarations: true
-BinPackParameters: false
-BreakBeforeBinaryOperators: false
-BreakBeforeBraces: Allman
-BreakBeforeTernaryOperators: false
-BreakConstructorInitializersBeforeComma: false
+--- 
+AccessModifierOffset: '-2'
+AlignAfterOpenBracket: AlwaysBreak
+AlignConsecutiveAssignments: 'false'
+AlignConsecutiveDeclarations: 'false'
+#AlignConsecutiveMacros: 'true'
+AlignEscapedNewlines: Right
+AlignOperands: 'false'
+AlignTrailingComments: 'false'
+#AllowAllArgumentsOnNextLine: 'false'
+#AllowAllConstructorInitializersOnNextLine: 'false'
+AllowAllParametersOfDeclarationOnNextLine: 'false'
+AllowShortBlocksOnASingleLine: 'false'
+AllowShortCaseLabelsOnASingleLine: 'false'
+AllowShortFunctionsOnASingleLine: Inline
+AllowShortIfStatementsOnASingleLine: 'true'
+#AllowShortLambdasOnASingleLine: 'true'
```

Comment:
We have commented this so it doesn't matter, but it should be.
    
    ```
    #AllowShortLambdasOnASingleLine: 'Inline'
    ```

Reply:
I will make that change at some point later.  That parameter was not recognized by the automatic format generator so it got commented out.